RELEASE_NOTES

```diff
@@ -1,6 +1,21 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.2.0 (26 November 2023)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Bug fixes:
+* none
+
+Improvements:
+* support for running tests inside the same job but after the build step (#222)
```

Comment:
We should mention `bot/test.sh` and `bot/check-test.sh` scripts that are picked up with the feature?

Reply:
Addressed in https://github.com/EESSI/eessi-bot-software-layer/pull/232/commits/c700a22c5b3db501b9bb58d18be0b37aa16b8182